<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChargeFace</class>
 <widget class="QDialog" name="ChargeFace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>后台管理</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:url(:/new/prefix1/bg.jpg);
border-radius:15px;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url();
font: 18pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>后台管理</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>381</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url();</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="searchData">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>331</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:#03a9f4;
color:#fff;
border-radius:15px;
};</string>
    </property>
    <property name="text">
     <string>查询数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changeData">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>331</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:#03a9f4;
color:#fff;
border-radius:15px;
}</string>
    </property>
    <property name="text">
     <string>处理房间信息</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="login_name">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url();
font: 9pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
